biogtr/training/test_batch_train.csv

```diff
@@ -0,0 +1,4 @@
+model.d_model,model.dim_feedforward,model.feature_dim_attn_head,model.num_encoder_layers,model.num_decoder_layers
```

Comment:
let's move this - could go in configs or just remove entirely. We could add a fixture for testing.